scripts/reboot.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -eu                                                                                                          
```

Comment:
This file is not used anywhere, OK to remove

Reply:
Yep! Thanks.